src/components/ebay-filter-menu/test/test.browser.js

```diff
@@ -0,0 +1,132 @@
+const { expect, use } = require('chai');
+const { render, fireEvent, cleanup } = require('@marko/testing-library');
+const { pressKey } = require('../../../common/test-utils/browser');
+const mock = require('./mock');
+const template = require('..');
+
+use(require('chai-dom'));
+afterEach(cleanup);
+
+/** @type import("@marko/testing-library").RenderResult */
+let component;
+
+describe('given the menu is in the default state', () => {
+    const input = mock.Basic_2Items;
+    const firstItemText = input.items[0].renderBody.text;
+    let footerButton, firstItem, secondItem;
+
+    beforeEach(async() => {
+        component = await render(template, input);
+        footerButton = component.getAllByRole('button')[0];
+        firstItem = component.getAllByRole('menuitemcheckbox')[0];
+        secondItem = component.getAllByRole('menuitemcheckbox')[1];
+    });
+
+    describe('when an item is clicked', () => {
+        beforeEach(async() => {
+            await fireEvent.click(component.getByText(firstItemText));
+        });
+
+        it('then it emits the filter-menu-change event with correct data', () => {
+            const selectEvents = component.emitted('filter-menu-change');
+            expect(selectEvents).to.length(1);
+
+            const [[eventArg]] = selectEvents;
+            expect(eventArg).has.property('el').with.text(firstItemText);
+            expect(eventArg).has.property('checked').to.deep.equal(['item 0']);
+        });
+    });
+
+    describe('when two items are clicked', () => {
+        beforeEach(async() => {
+            await fireEvent.click(firstItem);
+            await fireEvent.click(secondItem);
+        });
+
+        it('then it emits two menu-change events with correct data', () => {
+            const changeEvents = component.emitted('filter-menu-change');
+            expect(changeEvents).to.have.length(2);
+
+            const firstEventData = changeEvents[0][0];
```

Comment:
Maybe simplify this to:
`const [[firstEventData], [secondEventData] = changeEvents;`
OR
`const [firstEventData, secondEventData] = changeEvents.flat();`